web/router.ex

```diff
@@ -47,6 +47,8 @@ defmodule CoursePlanner.Router do
     resources "/tasks", TaskController do
       post "/grab", TaskController, :grab, as: :grab
     end
+
+    resources "/attendance", AttendanceController, only: [:index]
```

Comment:
the path of the resource `"/attendance"` is usually plural, could you change it please?